qa/acceptance/sys/ubuntu/user_bootstrap.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+##
+# Installing rbenv
+##
+#git clone git://github.com/sstephenson/rbenv.git .rbenv
+#echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bash_profile
+#echo 'eval "$(rbenv init -)"' >> ~/.bash_profile
+
+#git clone git://github.com/sstephenson/ruby-build.git ~/.rbenv/plugins/ruby-build
+#echo 'export PATH="$HOME/.rbenv/plugins/ruby-build/bin:$PATH"' >> ~/.bash_profile
+#source ~/.bash_profile
+
+
+##
+# install logstash from source code
+##
+#git clone https://github.com/elastic/logstash.git
+
+##
+# install jruby-1.7.24 and make it global
+##
+#rbenv install jruby-1.7.24
+#rbenv global  jruby-1.7.24
+
+#cd logstash
+#rake bootstrap
+#rake test:install-core
+
+cd
+wget https://download.elastic.co/logstash/logstash/packages/debian/logstash_2.3.1-1_all.deb
```

Comment:
this is now getting the latest 2.x branch released, however it should get it from local a shared dir as much as possible. The basic idea should be that:

* Pull last released version from URL.
* Push the generated version from local shared directory.

Will make sure to have this implemented next.

What do you think?